app/src/main/java/ch/hikemate/app/utils/MapUtils.kt

```diff
@@ -70,26 +72,27 @@ object MapUtils {
       true
     }
 
-    val startingMarker =
-        Marker(mapView).apply {
-          // Dynamically create the custom icon
-          icon =
-              createCircularIcon(
-                  context = mapView.context,
-                  fillColor = color,
-              )
```

Comment:
Warning: You didn't remove the `createCircularIcon`. Please remove the function as we won't use it anymore